misc/openapi-ballerina/modules/ballerina-to-openapi-generator/src/main/java/org/ballerinalang/ballerina/openapi/convertor/service/OpenApiResourceMapper.java

```diff
@@ -76,7 +76,7 @@
      * @param httpAlias    The alias for ballerina/http module.
      * @param swaggerAlias The alias for ballerina.swagger module.
      */
-    SwaggerResourceMapper(Swagger swagger, String httpAlias, String swaggerAlias) {
+    OpenApiResourceMapper(Swagger swagger, String httpAlias, String swaggerAlias) {
```

Comment:
can we change the parameter names to openApi

Reply:
Resolved with 81d9190